pygti/schemas.py

```diff
@@ -84,7 +95,74 @@
         "filter": [FilterEntry],
         "serviceTypes": [FilterServiceType],
         "useRealtime": bool,
+    },
+)
+
+SIRequest = Schema.extend(BaseRequestType, {"station": SDName})
+
+ContSearchByServiceId = Schema(
+    {
+        "serviceId": int,
+        "lineKey": str,
+        "plannedDepArrTime": GTITime,
+        "additionalOffset": int,
     }
 )
 
-SIRequest = Schema({"station": SDName})
+# kinds: GTI Docs say int but traffic on the website uses strings
+# Same thing with the value of the penalty. All are strings
```

Comment:
Can you add an example request with strings? I can't reproduce this. If sending integers works, the schema should have integers for validation (otherwise a value of "abc" would be considered valid)

Reply:
See example payloads in payloadsForTesting.py(l.277ff, 197ff):
```python
 "tariffInfoSelector": [
            {"tariff": "HVV", "tariffRegions": True, "kinds": ["1"]},
            {"tariff": "SH", "tariffRegions": False, "kinds": ["1"]},
        ],
```
These payloads are copied from requests i send using the webpage of the hvv